docs/standards/universal-profile/lsp0-erc725account.md

```diff
@@ -13,153 +13,235 @@ sidebar_position: 2
 
+### ERC1271
+
+:::tip
+
+See the **[ERC1271](https://eips.ethereum.org/EIPS/eip-1271)** standard for more information.
+
+Check the [**isValidSignature**](../smart-contracts/lsp0-erc725-account.md#isvalidsignature) function documentation.
+
+:::
+
+Unlike Externally Owned Accounts (EOAs), **smart contracts cannot sign messages** since they do not have private keys. This standard defines a way for contracts to verify if a signature and a message provided are valid according to the contract's logic. There will be many contracts that want to utilize signed messages to validate rights-to-move assets or other purposes.
+
+The **LSP0-ERC725Account** implements the **ERC1271** standard in a flexible way that allows for future upgradeability when different type of owners are set.
+
+When the owner of the account is an EOA, the **ECDSA algorithm** is used to recover the address of the signer from the provided signature and message, and the function `isValidSignature(..)` will return **valid** if the recovered signer address matches the address of the owner.
+
+When the owner is a smart contract, the `isValidSignature(..)` function will be called on the owner and return whether the signature and the message are **valid** according to the logic in `isValidSignature(..)` on the owner contract.
+
 ### LSP1 - UniversalReceiver
 
-:::info
+:::tip
 
 See the **[LSP1-UniversalReceiver](../generic-standards/lsp1-universal-receiver.md)** standard for more information.
 
+Check the [**universalReceiver functions**](../smart-contracts/lsp0-erc725-account.md#universalreceiver) provided by **LSP1** that allows notifying about incoming/ outgoing transactions.
+
+Check the **javascript** guides to know [**How to set a UniversalReceiverDelegate built by LUKSO**](../../guides/universal-receiver-delegate/set-default-implementation.md) or [**How to set your own UniversalReceiverDelegate**](../../guides/universal-receiver-delegate/accept-reject-assets.md).
+
 :::
 
 This standard enables the account to be notified of incoming transactions such as token transfer, vault transfer, information transfer, etc. Notifications are handy for situations where users want to customize how their account contract reacts to certain tokens by either rejecting them or operating a specific call on each token received.
 
 The **[LSP0-ERC725Account](#)** implements the `universalReceiver(..)` function that:
 
-- Emits an event with the typeId and data passed to it, as well as additional parameters such as the amount sent to the function, the caller of the function, and the return value of the delegate contracts.
+Emits an event with the `typeId` and `data` passed to it, as well as additional parameters such as the amount sent to the function, the caller of the function, and the return value of the delegate contracts.
```

Comment:
We could add a link to the `UniversalReceiver` event in the contract ABI page.
The we might not even need to mention what is emitted in the event in this page, since it is mentioned in the contract ABI page for the event (or keep it simple to a minimum).

```suggestion
Emits an [`UniversalReceiver`](./link-to-lsp0-contract-abi-page-UR-event) event with the `typeId` and `data`, as well as additional parameters such as the amount sent to the function, the caller of the function, and the return value of the delegate contracts.
```